Extended Project 3/Readme.md

```diff
@@ -0,0 +1,20 @@
+# Week 3: Extended Project
+## Web Server
+
+Build a simple HTTP server in Ruby
+
+## Instructions
```

Comment:
Thanks for the instructions.

Reply:
Is a requirement to present the project and quite helpful to order ideas and establish the nature and explain the development. Documenting isn't dead.